### DISCUSSION TITLE
MockK for Java project

### DISCUSSION THREAD

**Discussion by anandpc** [mockk/mockk#695] [Q&A] [answered]
Can we use MockK for a java based project ?
I was thinking if we can call java code from kotlin, then we may be able to use MockK as well to write unit tests for it. 
So can we do it ?
If yes, then are there any limitations ?

Thanks in Advance

**Comment by Raibaz** [comment, marked as answer]
Sure, you can.

There are some Java constructs that don't map particularly well to Kotlin (i'm thinking static fields and things like that), but generally speaking you can definitely use Mockk for testing a Java project :)
